### ISSUE TITLE
feature-request: use ./install-standalone when running ./install-profile

### ISSUE THREAD

**Issue by Clumsy-Coder** [ecarlson94/dotbot-template#7]
I noticed that the code in `./install-standalone` is closely similar to `./install-profile.`

The only difference being `./install-profile` is reading the `./meta/profiles/<profile>` contents and installing them in a loop later on.

![image](https://user-images.githubusercontent.com/19594044/164505131-c7185318-3fcb-4963-9c8a-d9e1cad7ab6d.png)

The execution of dotbot script is the same in `./install-standalone`.

Wouldn't it be better to to call `./install-standalone` script in `./install-profile` file in a for loop?
Something like:
```bash
#!/usr/bin/env bash

# install-profile

set -e

BASE_CONFIG="base"
CONFIG_SUFFIX=".yaml"

META_DIR="meta"
CONFIG_DIR="configs"
PROFILES_DIR="profiles"

DOTBOT_DIR="dotbot"
DOTBOT_BIN="bin/dotbot"

BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"


cd "${BASE_DIR}"
git submodule update --init --recursive


while IFS= read -r config; do
  CONFIGS+=" ${config}"
done < "${META_DIR}/${PROFILES_DIR}/$1"

shift

for config in ${CONFIGS} ${@}; do
  echo -e "\nConfigure $config"
  source ./install-standalone ${config}
done

cd "${BASE_DIR}"
```

or even somthing simple like:
```bash
#!/usr/bin/env bash

# install-profile

set -e

META_DIR="meta"
CONFIG_DIR="configs"
PROFILES_DIR="profiles"

BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"

cd "${BASE_DIR}"
git submodule update --init --recursive


while IFS= read -r config; do
  CONFIGS+=" ${config}"
done < "${META_DIR}/${PROFILES_DIR}/$1"

shift

source ./install-standalone ${CONFIGS}

cd "${BASE_DIR}"
```

Why do this?
- Less code to maintain
- Functionality already there

**Comment by kiriwalawren** [comment]
Sure. Why not?

I'm not sure when I will get to this though. I will try to complete it by the end of the weekend. Feel free to open up a PR if you like.

**Comment by Clumsy-Coder** [comment]
Feature implemented in #9
